pose_topic: /mavros/vision_pose/pose
twist_topic: /mavros/local_position/velocity_local
yaml_name: /config/vicon_param.yaml
message_timeout: 0.5
motor_speedup_time: 5.0
takeoff_height: 1.0
takeoff_land_speed: 0.3
reject_takeoff_twist_threshold: 0.15
hover_max_velocity: 3.0
hover_max_rate: 3.0
check_safety_volumn: true
safety_volumn: [-2.0, 2.0, -1.25, 2.25, -10, 2.5]
without_rc: false
throttle_channel: 3
yaw_channel: 4
pitch_channel: 2
roll_channel: 1
fsm_channel: 6
command_channel: 7
reboot_channel: 9
reverse_throttle: false
reverse_yaw: false
reverse_pitch: true
reverse_roll: false
switch_threshold: 0.75
joystick_deadzone: 0.10
check_centered_threshold: 0.25
hover_thrust: 0.204 #0.165
tau_phi: 0.1667
tau_theta: 0.1667
tau_psi: 0.4249434082976979